scripts/setup_sections/initial_setup_setup.sql

```diff
@@ -0,0 +1,3 @@
+-- initial_setup
+CREATE DATABASE test_django_db;
```

Comment:
delete this file, and create a database separately - e.g. by running `CREATE DATABASE test_django_db; CREATE DATABASE test_django_db_other` in `setup_module` function